packages/babel-preset-env/test/debug-fixtures/usage/stdout.txt

```diff
@@ -43,4 +43,6 @@ src/in.js -> lib/in.js
 [src/in2.js] Added following polyfills:
   regenerator-runtime { "chrome":"52", "firefox":"50", "ie":"11" }
   web.dom.iterable { "chrome":"52", "firefox":"50", "ie":"11" }
-src/in2.js -> lib/in2.js
\ No newline at end of file
+src/in2.js -> lib/in2.js
+🎉  successfully compil
```

Comment:
this seems off, the line got splitted into 2, some other `stdout.txt` suffer from the same thing, any idea why this happens?

Reply:
huh, that's just my regex not playing the way I liked. Weird that test didn't caught it (at least locally)?

Reply:
Oh, I was scoping my testing to `babel-cli` only, I'll update this shortly